.github/workflows/released_package_test.yml

```
@@ -0,0 +1,41 @@
+name: "released package test"
+
+on:
+  workflow_dispatch:
+    inputs:
+      delay_seconds:
+        description: "Delay in seconds before execution"
+        required: false
+        default: 0
+        type: number
+      version:
+        description: "Version to test"
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - name: sleep
+        run: |
+          echo "Starting after ${{ github.event.inputs.delay_seconds }} seconds"
+          sleep ${{ github.event.inputs.delay_seconds }}
+      - name: check @liam-hq/cli exit code
+        shell: bash
+        run: |
+          version="${{ github.event.inputs.version }}"
+          if [ -z "${version}" ]; then
+            echo "Error: Failed to extract version from package.json"
+            exit 1
+          fi
+          if [[ ! "${version}" =~ ^[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9.]+)?$ ]]; then
+            echo "Error: Invalid version format"
+            exit 1
+          fi
+          docker run --rm node:22-bullseye npx --yes @liam-hq/cli@${version} --version
```

What do you think of the following changes? 

```suggestion
          npx --yes @liam-hq/cli@${version} --version
```

It takes time to docker pull every time, and it should be similar to the environment in which the user runs the cli in CI.

---

Exactly!! 👀